<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestESDIg</class>
 <widget class="QMainWindow" name="TestESDIg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>852</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TestESDIg</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="txtbarriereEntree">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Barrière Entrée</string>
    </property>
   </widget>
   <widget class="QLabel" name="txtbarriereSortie">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>50</y>
      <width>181</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Barrière de sortie</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="barriereEntree">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>211</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="ArretBE">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>201</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Arrêt Barrière Entrée</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="MonteeBE">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>281</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Montée Barrière Entrée</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="DescenteBE">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>291</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Descente Barrière Entrée</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="barriereSortie">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>70</y>
      <width>211</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="ArretBS">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>201</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Arrêt Barrière Sortie</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="MonteeBS">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>281</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Montée Barrière Sortie</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="DescenteBS">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>291</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Descente Barrière Sortie</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="modifierAfficheur">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>270</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Modifier Afficheur</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>270</y>
      <width>48</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="txtposBarriereEntree">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>310</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Position Barrière Entrée</string>
    </property>
   </widget>
   <widget class="QLabel" name="txtposBarriereSortie">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>310</y>
      <width>241</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Position Barrière Sortie</string>
    </property>
   </widget>
   <widget class="QPushButton" name="niveauEntree">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Niveau Entrée</string>
    </property>
   </widget>
   <widget class="QPushButton" name="niveauSortie">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>420</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Niveau Sortie</string>
    </property>
   </widget>
   <widget class="QSlider" name="sliderEntree">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>350</y>
      <width>16</width>
      <height>160</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="sliderSortie">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>340</y>
      <width>16</width>
      <height>160</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="etatBPE">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>550</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Etat BPE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="etatBPS">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>550</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Etat BPS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="presenceVehiculeEntree">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>660</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>PresenceVehicule Entrée</string>
    </property>
   </widget>
   <widget class="QPushButton" name="presenceVehiculeSortie">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>670</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>PresenceVehicule Sortie</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridWidgetBT1" native="true">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>580</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayoutBT1">
     <item row="0" column="0">
      <widget class="QLabel" name="labelBT1">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridWidgetBT2" native="true">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>580</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayoutBT2">
     <item row="0" column="0">
      <widget class="QLabel" name="labelBT2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridWidgetBT3" native="true">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>710</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayoutBT3">
     <item row="0" column="0">
      <widget class="QLabel" name="labelBT3">
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridWidgetBT4" native="true">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>710</y>
      <width>321</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayoutBT4">
     <item row="0" column="0">
      <widget class="QLabel" name="labelBT4">
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
